StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -395,6 +409,16 @@ StETofGeomModule::addCounter( const TGeoPhysicalNode& gpNode, const float& dx, c
     mETofCounter.push_back( counter );
 }
 
+void
+StETofGeomModule::addCounter( const TGeoPhysicalNode& gpNode, const float& dx, const float& dy, const int moduleId, const int counterId, const double* safetyMargins, const StThreeVectorD alignment )
+{
+    StETofGeomCounter* counter = new StETofGeomCounter( gpNode, dx, dy, moduleId, counterId, alignment );
+
+    counter->setSafetyMargins( safetyMargins );
+
+    mETofCounter.push_back( counter );
+}
+
```

Comment:
This seems like a case where introducing a default argument is more appropriate. 

From C++ Style Guide https://google.github.io/styleguide/cppguide.html#Default_Arguments:

"Compared to overloading the function, default arguments have a cleaner syntax, with less boilerplate and a clearer distinction between 'required' and 'optional' arguments."